client/highlevel-rest/build.gradle

```diff
@@ -36,26 +31,30 @@ dependencies {
   testCompile "org.hamcrest:hamcrest-all:${versions.hamcrest}"
 }
 
-dependencyLicenses {
-  // Don't check the client's license. We know it.
-  dependencies = project.configurations.runtime.fileCollection {
-    it.group.startsWith('org.elasticsearch') == false
+task checkNoOtherDependencies {
```

Comment:
I'm just afraid that someone blindly copy/paste a new dependency without looking at the comment :/